src/functions/build_dependency_packages.py

```diff
@@ -20,19 +20,37 @@ def build_dependency_packages(mpr_rpc_json_data, resultcount):
 
 		depends_packages = []
 
-		for j in ['depends', 'makedepends', 'checkdepends']:
+		if j in [f'{os_codename}_conflicts', f'{os_codename}_makedepends', f'{os_codename}_checkdepends']:
 			depends_packages += get_srcinfo_value(j, True)
+		else:
+			for j in ['depends', 'makedepends', 'checkdepends']:
+				depends_packages += get_srcinfo_value(j, True)
+
+
+		conflicts_packages_ds = get_srcinfo_value((os_codename + '_conflicts'), False)
+		replaces_packages_ds = get_srcinfo_value((os_codename + '_replaces'), False)
+		breaks_packages_ds = get_srcinfo_value((os_codename + '_breaks'), False)
+		provides_packages = get_srcinfo_value((os_codename + '_provides'), False)
```

Comment:
Going off of the previous comment, I'd like these to be labeled like `distro_conflicts_packages` (and then likewise update the relevant variables below).

I'd also like these to use `f"{os_codename}_conflicts"` instead of the `os_codename + '_conflicts'` syntax (and then likewise remove the parenthesis that are surrounding them).